apitools/base/py/transfer.py

```diff
@@ -344,7 +344,9 @@ def __GetChunk(self, start, end=None, additional_headers=None):
         self.EnsureInitialized()
         end_byte = end
         if self.total_size and end:
-            end_byte = min(end, self.total_size)
+            end_byte = min(end_byte, self.total_size)
+        # if self.chunksize > 0:
```

Comment:
In the _Transfer base class, we accept a chunksize kwarg and use it like so:
self.chunksize = chunksize or 1048576

Which means a chunksize of 0 as a kwarg won't be respected.